src/milia/api/user.cljc

```
@@ -21,6 +22,13 @@
         response (parse-http :get url :suppress-4xx-exceptions? true)]
     (if-let [error (:detail response)] nil response)))
 
+(defn get-profiles-for-list-of-users
+  "Return the profile for the account username or the passed username."
+  [users]
+  (let [url (make-url (str "profiles" "?users=" (join "," users)))
+        response (parse-http :get url :suppress-4xx-exceptions? true)]
+     (if-let [error (:detail response)] nil response)))
```

Shouldn't we return the error instead of a `nil` response?
i.e.
```clojure
(if-let [error (:detail response)] error response)))
```